docs/development/building-good-containers.md

```diff
@@ -60,7 +60,7 @@ as not being able use the `NODE_EXTRA_CA_CERTS` environment
 variable. You can read more about the issue
 [here](https://github.com/nodejs/node/pull/37727).
 
-Typical work arounds which include using firewall functionality to
+Typical work around which include using firewall functionality to
```

Comment:
This one was correct and should not be changed.

Reply:
thanks!